Changes

```diff
@@ -1,5 +1,12 @@
 Revision history for Perl extension Net::SSLeay.
 
+????
+	- Create SSL_CTXs with Test::Net::SSLeay's new_ctx() function for tests that
+	  are broken with LibreSSL 3.2. Partially fixes GH-232.
+	- In 36_verify.t, account for the presence of the X509_V_FLAG_LEGACY_VERIFY
+	  flag (signalling the use of the legacy X.509 verifier) in LibreSSL 3.2
+	  versions from 3.2.4 onwards. Fixes the remainder of GH-232.
+
```

Comment:
Suggestion: Add a pointer to updated SSLeay.pod _KNOWN BUGS AND CAVEATS_ section that now discusses LibreSSL 3.2 compatibilty in detail.

Changes is likely a file that is checked by many to quickly see what's new, therefore it would be useful to point people to newly updated information too.